app/frontend/helpers/isEpochBoundaryUnderway.ts

```diff
@@ -0,0 +1,16 @@
+import * as moment from 'moment'
+import {EPOCH_TRANSITION_MAX_MINUTES} from '../constants'
```

Comment:
I'd consider just hardcoding the constant, I don't really see it being reused and whoever inspects this code would just have the overhead of having to look up the constant, and whoever inspects the constants file would wonder, what's that constant for even though it's rather unimportant compared to others